src/main/java/com/cvsgo/entity/User.java

```diff
@@ -90,4 +89,12 @@ public void updateRoleToRegular() {
         this.role = Role.REGULAR;
     }
 
+    public void updateUser(User user, String nickname, List<Tag> tags) {
+        this.nickname = nickname;
+        user.userTags.clear();
+        for (Tag tag : tags) {
+            user.addTag(tag);
```

Comment:
현재 클래스가 User 클래스인데 User 객체를 받아서 처리할 필요가 없을 것 같아요. nickname에 접근하신 것처럼 this.userTags 또는 그냥 userTags로 접근하시면 될 것 같습니다~
```suggestion
    public void updateUser(String nickname, List<Tag> tags) {
        this.nickname = nickname;
        this.userTags.clear();
        for (Tag tag : tags) {
            this.addTag(tag);
```